wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Customer.kt

```diff
@@ -0,0 +1,64 @@
+package org.stellar.walletsdk.customer
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import okhttp3.MediaType.Companion.toMediaType
+import org.stellar.walletsdk.auth.AuthToken
+import org.stellar.walletsdk.json.toJson
+
+const val APPLICATION_JSON_CHARSET_UTF_8 = "application/json; charset=utf-8"
+val TYPE_JSON = APPLICATION_JSON_CHARSET_UTF_8.toMediaType()
+
+class Customer
+internal constructor(
+  private val token: AuthToken,
+  private val baseUrl: String,
+  private val httpClient: HttpClient
+) {
+  /**
+   * Get customer information by customer ID.
+   *
+   * @param id customer id
+   * @return a customer data object
+   */
+  suspend fun getById(id: String): CustomerData {
+    val urlString = "$baseUrl/sep12/customers/id=$id"
+
+    val response = httpClient.get(urlString) { header(HttpHeaders.Authorization, "Bearer $token") }
+
+    return response.body<CustomerData>()
+  }
+
+  /**
+   * Create a new customer.
+   *
+   * @param customer map of customer information
+   * @return a customer with id information
+   */
+  suspend fun add(customer: Map<String, String>): AddCustomerResponse {
```

Comment:
Can you separate sep-9 fields (map) and sep-12 fields (such as id, type)

Reply:
Done